hvplot/tests/testoperations.py

```diff
@@ -324,6 +319,23 @@ def test_downsample_resample_when(self, kind, eltype):
         assert isinstance(element, eltype)
         assert len(element) == 0
 
+    @parameterized.expand([(None,), (True,), ('vline',), ('hline',)])
```

Comment:
Many more tests will be needed to cover all the branches of the code introduced in the converter, these first two are a good start but they're very basic and just check whether the code doesn't error and the HoloViews type is the expected one.